Rewrite the Tekton Triggers installation instructions for clarity and flow (#966)
tektoncd/triggers: docs/install.md

<!--
---
linkTitle: "Setting Up Tekton Triggers"
weight: 2
---
-->
# Setting Up Tekton Triggers

Follow the steps below to set up an official release of Tekton Triggers on your Kubernetes cluster. 
If you want to test Triggers from `HEAD`, see the
[Tekton Developer Guide](https://github.com/tektoncd/triggers/blob/master/DEVELOPMENT.md#install-triggers).

## Prerequisites

You must satisfy the following prerequisties to set up Tekton Triggers:

> * Your cluster must be running Kubernetes version 1.18 or above.

Reviewer: ```suggestion
* You must have a Kubernetes cluster running version 1.18 or later.
```

I would also be in favor of keeping the `kind` instructions as they match `pipeline` install guides:
https://github.com/tektoncd/pipeline/blob/master/docs/install.md#before-you-begin

Author: I want to avoid duplication of instructions wherever possible because they can quickly go out of sync between all their different locations. Plus, they must have Pipelines running before installing Triggers, they can't run Triggers without Pipelines, so either way they must go look at Pipelines installation instructions.

Author: Restored the `kind` instructions as per review comments.

- [Installing Tekton Triggers](#installing-tekton-triggers-1)
## Prerequisites

## Pre-requisites
You must satisfy the following prerequisties to set up Tekton Triggers:

1.  A Kubernetes cluster version 1.18 or later
* You must have Tekton Pipelines installed on your Kubernetes cluster. For instructions, see [Installing Tekton Pipelines](https://github.com/tektoncd/pipeline/blob/master/docs/install.md).

    If you don't already have a cluster, you can create one for testing with

>     `kind`.  [Install

Reviewer 2: I think it is helpful to still briefly mention kind here in this context. imo, it helps ensure we have a path with low barriers to contribution and get folks on a common path.

Author: Restored the `kind` instructions as per review comments.

> * Your Kubernetes cluster must be running Kubernetes version 1.18 or above.

Reviewer 3: I'd mention this before the point about installing Tekton Pipelines. And I'd keep the bit about being able to create a cluster easily using Kind. It does make things a bit more verbose but IMO makes the docs more approachable for someone reading this for the first time.

Author: Restored the `kind` instructions as per review comments and moved the Pipelines requirement after the Kubernetes cluster requirement.

> 1. Open a terminal on your cluster and log in as the user with which you installed Tekton Pipelines.

Reviewer: This seems a bit misleading to me. You don't open a terminal on your cluster technically. You mainly need a `kubeconfig` which is pointing at the cluster with the correct role.

Author: Fair! Reworded.
